### ISSUE TITLE
Cannot read property '0' of undefined 

### ISSUE THREAD

**Issue by guillemcordoba** [Holo-Host/cross-origin-message-bus#18]
When calling a zome function, this error occurs:

```
inline-module-index-1.js:9053 Uncaught TypeError: Cannot read property '0' of undefined
    at ParentAPI.<anonymous> (inline-module-index-1.js:9053)
    at ParentAPI.listener (inline-module-index-1.js:8904)
```

following the trail it's because of this line: https://github.com/Holo-Host/cross-origin-message-bus/blob/86c3fb99ba1d1afe230dc2a9fa356ed43b4714d0/src/index.ts#L205, where data seems to be undefined.

This is a bit weird, because the app continues to work, and all the zome functions that we call do return something.

**Comment by timotree3** [comment]
I am guessing the exception is on this line https://github.com/Holo-Host/cross-origin-message-bus/blob/86c3fb99ba1d1afe230dc2a9fa356ed43b4714d0/src/index.ts#L338

**Comment by timotree3** [comment]
I don't think the exception can come from the destructuring assignment you linked because in Chromium, destructuring `undefined` gives:

```
Uncaught TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
```

and indexing `undefined` gives
```
Uncaught TypeError: Cannot read property '0' of undefined
```

**Comment by timotree3** [comment]
Gotcha. That seems to imply that `.emit("response", undefined)` is called on a `ParentAPI` somewhere, but every single instance of "response" is accompanied by an array of length 2. I don't see how this error could possibly be happening

**Comment by guillemcordoba** [comment]
Yeah I don't know... I did a console.log on data and it never showed undefined. But still this is what I'm getting... Let's see if others get the same when this becomes more used.
